content/project-news/reactos-0414-released.md

```diff
@@ -0,0 +1,119 @@
+
+## ICMP improvements
+[Internet Control Message Protocol](https://en.wikipedia.org/wiki/Internet_Control_Message_Protocol) (or ICMP by short) is a protocol used by network device drivers to send network information such as errors or log status.
+The implementation was rather scarce and incomplete with pieces of code unimplemented or unfinished.
+The IP Helper module primarily uses this protocol and as a matter of that, it brought a lot of issues and bugs. 
+
+Thanks to Victor Perevertkin, and with the help of Tim Crawford, the ICMP protocol implementation has seen lots of improvements.
+These range from the implementation of the IOCTL_ICMP_ECHO_REQUEST I/O control code to a full  rewrite of the Icmp** routines.
+As a result, the network device drivers can now handle ICMP requests in a correct manner.
+This is proven in the screenshot below, which shows the tracert command-line utility now outputting complete information.
+
+{{< figure link="/img/project-news/reactos-0414-released/icmp.png" src="/img/project-news/reactos-0414-released/icmp.png" >}}
+
+## Kernel improvements
+The NT kernel of ReactOS is the heart that is responsible for the correct functioning of the operating system and interfacing with the bare metal.
+The stability of the system drastically relies on the robustness of the kernel.
+Every bug counts here.
+
+With every release, the developers squash bugs in the kernel to offer a better overall stability and experience.
+ReactOS 0.4.14 comes with several improvements to the Memory Manager thanks to Thomas Faber.
+Speaking of the aforementioned module, Vadim Galyant did some initial work on PAE (Physical Address Extension) and contributed towards the Memory Manager sections rewrite.
+
+Timo Kreuzer added CRT (C Run-Time) exception handling by importing related code from WINE. This allows 64-bit development to proceed as various bugs and random hangs on 64-bit systems have been resolved.
+Victor Perevertkin fixed a bug in storage class PnP driver (classpnp.sys) 
+and both Timo Kreuzer and Thomas Faber fixes some buffer overflows vulnerabilities. To finish with, Eric Kohl improved the device action worker code in I/O.
+
+Let's not forget the PnP (Plug n' Play) manager of the kernel which is also an important part not only in regard of stability the usability of the system to a plethora of hardware.
+Vadim Galyant added the initial headers implementation of the Resources Arbiter library which can be used by the bus
+drivers like ACPI, PCI or legacy PIC HAL. In addition to that Vadim implemented debug code both in PnP and I/O managers whereas Eric Kohl improved the PnP manager to map device capabilities to capability flags. Thomas Faber fixed a critical bug that could
+cause memory corruption in the kernel space and fixed a bad IRP (I/O request packet) handling that caused a BSoD (Blue Screen of Death) on first generation Xbox console machine with USB enabled. 
```

Comment:
```suggestion
cause memory corruption in the kernel space and fixed a bad IRP (I/O request packet) handling that caused a BSoD on an Xbox console with USB enabled. 
```

Reply:
I think the explanation for BSoD (Blue Screen of Death) was good for average joe.
And specifically specifying the **first generation** XBox was most likely intentionally

Reply:
indeed, because for some people, Xbox could mean the 360.